### PR TITLE
update locate db & hide/show hidden files

### PR DESCRIPTION
**Pull request by selfagency**
Added two functions: One to update the 'locate' database so one may conduct command line file searches; and one to reveal and conceal so-called hidden files in the Finder.

Signed-off-by: Daniel Sieradski daniel@self.agency

### CODE REVIEW

File: showfiles.fish [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function showfiles -d "Reveals and conceals hidden files in Finder"
+  if [ (count $argv) -gt 0 ]
```
Comment (bpinto) [inline showfiles.fish:2]: Shouldn't we be checking if it's YES or NO?

Comment (selfagency) [inline showfiles.fish:2]: yeah, sorry, i'm new to fish shell scripting. will work on it.

Comment (bpinto) [inline showfiles.fish:2]: :+1:

Good work!

Comment (selfagency) [inline showfiles.fish:2]: thanks

**Pull request closed without merging**